src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -267,13 +293,24 @@ private void writeVariants(JavaRDD<VariantContext> variants) {
             final Broadcast<ReferenceMultiSource> reference,
             final Broadcast<HaplotypeCallerArgumentCollection> hcArgsBroadcast,
             final ShardingArgumentCollection assemblyArgs,
-            final SAMFileHeader header) {
+            final SAMFileHeader header,
+            final Broadcast<VariantAnnotatorEngine> annotatorEngineBroadcast) {
```

Comment:
I can't remember, is the reason we don't pass the engine in as a broadcast here because you cant serialize / broadcast the reference source?